data-raw/standard-format-data-prep/README.md

```diff
@@ -20,7 +20,8 @@ This directory contains the following Rmd documents:
 -   [`rst_environmental_standard_format.Rmd`](https://github.com/FlowWest/JPE-datasets/blob/documentation/data-raw/standard-format-data-prep/rst_environmental_standard_format.Rmd) contains the code used to generate `standard_environmental.csv` which contains environmental covariate measurements taken during trap visits.
 -   [`flow_standard_format.Rmd`](https://github.com/FlowWest/JPE-datasets/blob/documentation/data-raw/standard-format-data-prep/flow_standard_format.Rmd) contains the code used to generate `standard_flow.csv` which compiles flow data from publically available streamgages.
 -   [`adult_upstream_passage_standard_format.Rmd`](https://github.com/FlowWest/JPE-datasets/blob/documentation/data-raw/standard-format-data-prep/adult_upstream_passage_standard_format.Rmd) contains the code used to generate `standard_adult_upstream_passage.csv`
--   In Progress: adult holding, redd, carcass, temperature
+-   [`holding_standard_format.Rmd`] (https://github.com/FlowWest/JPE-datasets/blob/documentation/data-raw/standard-format-data-prep/holding_standard_format.Rmd) contains the code used to generate `standard_holding.csv`
+-   In Progress: adult redd, carcass, temperature
 
 Data descriptions are included within each Rmd. Each standard dataset is described in more detail below.
```

Comment:
The documentation stuff all looks good. Once all the standard format things are done and we migrate the package into the JPE org we can do one more pass through of all the doc stuff